README.md

```diff
@@ -157,6 +157,7 @@ API | Description | Auth | HTTPS | CORS | Link |
 API | Description | Auth | HTTPS | CORS | Link |
 |---|---|---|---|---|---|
 | Binance | Exchange for Trading Cryptocurrencies based in China | `apiKey` | Yes | Unknown | [Go!](https://github.com/binance-exchange/binance-official-api-docs) |
+| BitBNS | Indian Cryptocurrency exchange price ticker | No | Yes | No | [Go!](https://bitbns.com/order/getTickerWithVolume/) |
```

Comment:
Please link to documentation rather than a live endpoint.

Reply:
They do not have a Official documentation